Airship/AirshipCore/Source/WorkRateLimiterActor.swift

```diff
@@ -33,14 +33,20 @@ actor WorkRateLimiter {
     }
 
     func nextAvailable(_ keys: [String]) -> TimeInterval {
```

Comment:
I wonder if we can get away with:
```
 return keys.reduce(0.0) { maxDelay, key in
            guard case .overLimit(let delay) = status(key) else {
                return maxDelay
            }
            return max(maxDelay, delay)
        }
```

instead? I believe it wont create another array

Reply:
Nice. Much cleaner. Let me update. I also want to add some tests